codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoWriter.java

```diff
@@ -269,10 +269,23 @@ public String apply(Object type, String indent) {
             if (type instanceof Symbol) {
                 Symbol typeSymbol = (Symbol) type;
                 addUseImports(typeSymbol);
-                if (isExternalNamespace(typeSymbol.getNamespace())) {
-                    return formatWithNamespace(typeSymbol);
+
+                Boolean isUniverseType = typeSymbol.getProperty(SymbolUtils.GO_UNIVERSE_TYPE, Boolean.class)
+                        .orElse(false);
```

Comment:
is `UniverseType` for builtins?

Reply:
Yes, the naming choice of "universe" comes from the Go specification https://golang.org/ref/spec#Predeclared_identifiers

Reply:
Could you put that in a comment